scrapers/ExCoGi.yml

```diff
@@ -0,0 +1,32 @@
+name: "ExCoGi"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - exploitedcollegegirls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class='info']
+    scene:
+      Title: $info/h4/strong/text()
+      Details: //article[@id='about']/p/text()
```

Comment:
some scenes have a lot of paragraphs eg `https://exploitedcollegegirls.com/free/scene/487/1339/ecg-video-taylor`
```yaml
      Details:
        selector: //article[@id='about']/p/text()
        concat: "\n"
```